projects/tempest-finance/index.js

```diff
@@ -0,0 +1,20 @@
+const chains = require("./chains");
+const {sumTokens} = require("../helper/sumTokens");
+
+async function tvl(api, chain) {
```

Comment:
you can skip chain param, and pull it from api as `api.chain`

Reply:
Updated